implementations/rust/ockam/ockam/src/lib.rs

```diff
@@ -6,6 +6,9 @@
 #[macro_use]
 extern crate serde_big_array;
 
+#[macro_use]
+extern crate tracing;
```

Comment:
This is not required

Reply:
There is a `trace!` line in the protocol parser and I think we will want to start logging more in the ockam crate, which is made much more easier by including the macros in the root of the ockam crate

Reply:
I thought we're moving towards latest rust syntax https://doc.rust-lang.org/edition-guide/rust-2018/macros/macro-changes.html
Do you think #[macro_use] is more convenient?

Reply:
For stuff like the log macros I think `#[macro_use]` is more convenient, because otherwise you have to import all different log levels you want to use in a file, so `use tracing::{info, warn}`, and say you want to change a `warn` to `error`, then you need to add the import, etc